float32 grasping_force
---
bool ans
